src/include/btree.i

```diff
@@ -1189,7 +1189,7 @@ __wt_page_las_active(WT_SESSION_IMPL *session, WT_REF *ref)
 
 	if ((page_las = ref->page_las) == NULL)
 		return (false);
-	if (page_las->invalid || !ref->page_las->skew_newest)
+	if (!ref->page_las->skew_newest)
```

Comment:
We can strip the `ref->` from this -- I'm sure it makes no difference, but it read oddly.